pep/app/services/current-user.ts

```diff
@@ -1,10 +1,10 @@
 import Service, { inject as service } from '@ember/service';
+import { assign } from '@ember/polyfills';
```

Comment:
@billdami I do not believe you need to use assign from `@ember/polyfills`. It looks like all browsers we support have supported `Object.assign` since 2015

https://caniuse.com/?search=object.assign